packages/@aws-cdk/pipelines/README.md

```diff
@@ -749,6 +749,18 @@ ID: ...)
 The stack failed its previous deployment, and is in a non-retryable state.
 Go into the CloudFormation console, delete the stack, and retry the deployment.
 
+### Cannot find module 'xxxx' or its corresponding type declarations.
```

Comment:
Doesn't sound isolated to CDK pipelines.

Should this go here instead? https://docs.aws.amazon.com/cdk/latest/guide/troubleshooting.html

Reply:
It's not strictly "isolated to", no.

But I predict it will be more prevalent in CDK pipelines because the CodeBuild environment is less strictly controlled by users (it's a default image provided by CodeBuild).

Still makes sense to put it close to where people are likely to encounter it.